receiver/hostmetricsreceiver/internal/scraper/networkscraper/config.go

```diff
@@ -14,9 +14,24 @@
 
 package networkscraper
 
-import "go.opentelemetry.io/collector/receiver/hostmetricsreceiver/internal"
+import (
+	"go.opentelemetry.io/collector/internal/processor/filterset"
+	"go.opentelemetry.io/collector/receiver/hostmetricsreceiver/internal"
+)
 
 // Config relating to Network Metric Scraper.
 type Config struct {
 	internal.ConfigSettings `mapstructure:",squash"` // squash ensures fields are correctly decoded in embedded struct
+
+	// Include specifies a filter on the network interfaces that should be included from the generated metrics.
+	// Exclude specifies a filter on the network interfaces that should be excluded from the generated metrics.
```

Comment:
This comment should be next to `Exclude` field, and the line below will need to be duplicated probably.